src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnNotSpecified_Expected.sarif

```diff
@@ -1,46 +1,43 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-1.0.0",
-  "version": "1.0.0",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0",
+  "version": "2.0.0",
   "runs": [
     {
-      "id": "dc4a37c2-19b8-4db5-8d4b-93016a5b34ba",
       "tool": {
-        "name": "SarifCli",
-        "fullName": "SarifCli 1.0.0.0",
-        "version": "1.0.0.0",
-        "semanticVersion": "1.0.0",
-        "sarifLoggerVersion": "1.5.31.0",
+        "name": "Sarif.Multitool",
+        "fullName": "Sarif.Multitool 2.0.0.0",
+        "version": "2.0.0.0",
+        "semanticVersion": "2.0.0",
+        "sarifLoggerVersion": "2.0.0.0",
         "language": "en-US",
         "properties": {
-          "Comments": "Command line tool for working with SARIF log files.",
+          "Comments": "Command line tool to perform transformations of input files to SARIF.",
           "CompanyName": "Microsoft Corporation",
           "ProductName": "Microsoft SARIF SDK"
         }
       },
-      "toolNotifications": [
+      "results": [],
+      "invocations": [
         {
-          "id": "MSG001.AnalyzingTarget",
-          "physicalLocation": {
-            "uri": "file:///G:/Code/sarif-sdk/src/Microsoft.CodeAnalysis.Sarif.Multitool/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnNotSpecified.sarif"
-          },
-          "message": "Analyzing 'EndColumnNotSpecified.sarif'...",
-          "level": "note",
-          "time": "2016-08-29T22:59:10.501Z"
+          "startTime": "2018-07-25T16:16:41.527Z",
+          "endTime": "2018-07-25T16:16:42.173Z",
```

Comment:
>07-25T16:16:42.173Z [](start = 27, length = 19)

same note. your matching is resilient to this non-deterministic data, but git is not: every time we run your script, we will generate a new timestamp. other tools use regex to strip this data out. might be some code in the binskim tree if it isn't in SARIF. search the powershell scripts for 'endTime'

Reply:
Hmm, that's true -- it does complicate the history. Do we have a "--deterministic" option on the Driver?

---
In reply to: [205219160](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205219160) [](ancestors = 205219160)

Reply:
found it. search for endTime. no one is proud of this code. if you improve on it or add other non-deterministic things, let me know so we can update binskim. also note: i will shortly add an option to strip all non-deterministic data, so this may all be a wash in a couple of weeks anyway. so you might want to just grab this stuff in the meantime or frankly ignore the note. your instincts will be sound on how to proceed i am sure.

[https://github.com/Microsoft/binskim/blob/master/src/Test.FunctionalTests.BinSkim.Driver/UpdateBaselines.ps1](https://github.com/Microsoft/binskim/blob/master/src/Test.FunctionalTests.BinSkim.Driver/UpdateBaselines.ps1)

---
In reply to: [205219160](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205219160) [](ancestors = 205219160)

Reply:
If you're in the process of adding an option (I assume you mean a new Multitool verb, such as "Sarif.Multitool make-deterministic", then I will leave this as is.

---
In reply to: [205220055](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205220055) [](ancestors = 205220055,205219160)

Reply:
Sagacious.

---
In reply to: [205221602](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205221602) [](ancestors = 205221602,205220055,205219160)